Sources/Models/GitHub.swift

```diff
@@ -39,18 +40,21 @@ func githubRepository(from url: URL) -> Result<GitHubRepository, GeneratePlistEr
 
     guard let host = url.host,
         host.contains(gitDomain),
-        url.pathComponents.count >= 2,
-        url.pathComponents[url.pathComponents.count - 1].hasSuffix(gitSuffix) else { return .failure(.unknownRepository) }
+        url.pathComponents.count >= 2
+    else { return .failure(.unknownRepository) }
 
+    let name = url.pathComponents.last!.contains(gitSuffix)
+        ? String(url.pathComponents[url.pathComponents.count - 1].dropLast(gitSuffix.count))
+        : String(url.pathComponents[url.pathComponents.count - 1].replacingOccurrences(of: gitSuffix, with: ""))
```

Comment:
In the motto of ,better safe than sorry', we could use `[safe: ]` subscript every time we use array subscript in this file.

Reply:
I think we can also use the `lastPathComponent` then which reads even nicer imo.
for the subscript: I added one to Helpers now (or is there something I can use already. Swift itself doesn't offer that, right?) 
Added an additional guard but not sure if throwing the correct error?